lib.sh

```diff
@@ -1080,6 +1080,31 @@ case "${1}" in
 esac
 }
 
+# Example:
+# notify_user_gui \
+# "Subject"
+# "Message"
+#
+# occ_command notification:generate -l "$2" "$admin" "$1"
+notify_user_gui() {
+USER=$(occ_command user:list | awk '{print $2}' | cut -d ":" -f1;)
+print_text_in_color "$ICyan" "Looping through users..."
+for user in $USER
+do
+    if occ_command user:info "$user" | grep -q "admin";
```

Comment:
actually I would grep  for "- admin"
since it is possible theoratically that a user as admin in it, but  "- admin" definitely grep's the group.